docs/tutorials.rst

```diff
@@ -250,41 +258,39 @@ your application.
 .. code-block:: python
    :linenos:
 
-    import random
-    import string
-
-    import cherrypy
-
-    class StringGenerator(object):
+   import random
```

Comment:
Currently it's impossible to copy, paste and run code snippets of `tut05.py` up to `tut09.py`, because there is a one preceding space in each line. You have to manually remove space character in every line of source snippets of tutorial. `tut01.py` up to `tut04.py` uses three space indent.